client/gutenboarding/create-site.tsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+
+import { Header } from '@wordpress/edit-post';
+import { SelectControl } from '@wordpress/components';
+import Main from 'components/main';
+
+export function CreateSite() {
+	const [ siteType, setSiteType ] = useState( 'blog' );
+
+	return (
+		<>
+			<Header />
+			<Main>
+				I want to create a <u>website</u>
+				<SelectControl
+					value={ siteType }
+					onChange={ setSiteType as ( nextState: string ) => void }
+					options={ [
+						{ label: 'with a blog', value: 'blog' },
+						{ label: 'for a store', value: 'store' },
+						{ label: 'to write a story', value: 'story' },
+					] }
+				/>
+				.
+			</Main>
```

Comment:
Do we clean this up a little before committing? AFAICT there's no `header` export in edit-post (?). This is what shows on my screen when cleaning up the error:

<img width="911" alt="Screenshot 2019-10-10 at 4 11 16 PM" src="https://user-images.githubusercontent.com/1705499/66571837-9fbbd200-eb78-11e9-903e-1ad48222329c.png">

Reply:
@chriskmnds Out of curiosity, how did you clean up the error? :grimacing:

Reply:
just removed header and the import? 😀